distribution/build.gradle

```diff
@@ -549,6 +549,14 @@ subprojects {
       'license.text': [
         'deb': licenseText,
       ],
+
+      'keystore.passwd.check': [
+        /* Avoid unfortunate 5-second delay for elasticsearch-cli
+           scripts on certain OSX/JVM combos by reading the
+           "encrypted" flag byte in the keystore directly */
+        'tar': '[ "$(echo -n "\000")" = "$(dd if="$ES_PATH_CONF"/elasticsearch.keystore ibs=1 skip=31 count=1 2>/dev/null)" ]',
```

Comment:
I don't think should work around this and complicate our scripts in the process. Rather, I think we should expect that a user has setup their system to avoid this problem.

Reply:
I agree that this is a bad workaround. Not only is it ugly and unclear, it's also got the potential to become incorrect when the format of the keystore changes. It's much better to be able to delegate to the Keystore CLI tools.

However, given the potential for this to give some percentage of our users the bad experience of an unexpected delay in the main command for the application, I wanted to provide the best workaround I could think of in this work-in-progress PR.

If anyone is strongly in favor of this workaround, please let me know. (@mark-vieira - do you have any thoughts?) Otherwise, I will plan to back this change out and look for some place under troubleshooting or FAQs to document the 5-second-delay bug/issue.

Reply:
The bug you refer to affects _any_ java application that calls `InetAddress.getLocalHost()`. Fixing it here might very well not do us any good if that's needed somewhere else down the line. I'd say the CLI latency is at most annoying. Folks will unlikely be calling this as much as you were during development 😉

Reply:
In particular, if a user is impacted by this, it occurs when starting Elasticsearch or practically any other CLI tool. Given the impact, a user impacted by this would reach out for support quickly. We don't see frequent reports of this issue, so I don't think it's a widespread problem. Given that, I would rather that we not add to the docs verbiage around this as otherwise we can end up with overload in the docs.